docs/dev/guide/hello-world.md

```diff
@@ -609,6 +497,105 @@ async changeGreeting() {
 },
 ```
 
+You now have a fully functioal Greeter-dApp! However, it does not leverage any zkSync-specific features.
+
+### Paying fees using testnet paymaster
+
+Even though ether is the only token you can pay fees with, the account abstraction feature allows you to integrate [paymasters](../zksync-v2/aa.md#paymasters) that can either pay the fees entirely for you or swap your tokens on the fly. In this tutorial, we will use the [testnet paymaster](../zksync-v2/aa.md#testnet-paymaster) that is provided on all zkSync testnets. It allows users to pay fees in an ERC20 token with the exchange rate of ETH of 1:1, i.e. one unit of the token for one wei of ETH.
+
+::: tip Mainnet integration
+
+Testnet paymaster is purely for demonstration of the feature and won't be available on mainnet. When integrating your protocol on mainnet, you should follow the documentation of the paymaster you will use.
+
+:::
+
+The address of the paymaster as well as the required input should be provided in the `getOverrides` method.
+
+1. We need to retrieve the address of the testnet paymaster:
+
+```javascript
+async getOverrides() {
+  if (this.selectedToken.l1Address != ETH_L1_ADDRESS) {
+    const testnetPaymaster = await this.provider.getTestnetPaymasterAddress();
+
+    // ..
+  }
+
+  return {};
+}
+```
+
+Note, that it is recommended to retrieve the testnet paymaster's address each time before any interactions as it may change.
+
+2. Add `utils` to the imports from `zksync-web3` SDK:
+
+```javascript
+import { Contract, Web3Provider, Provider, utils } from "zksync-web3";
+```
+
+2. We need to calculate how many tokens are required to process the transaction. Since the testnet paymaster exchanges any ERC20 token to ETH at 1:1 rate, then the number is the same as of ETH:
```

Comment:
```suggestion
2. We need to calculate how many tokens are required to process the transaction. Since the testnet paymaster exchanges any ERC20 token to ETH at a 1:1 rate, the number is the same as the ETH value:

Reply:
"ETH value" is a bad phrasing, since there is also `value` field which denotes the number of ETH passed with the transaction (not the fee)

Reply:
edited